doc/en/user/source/data/raster/imagemosaic/configuration.rst

```diff
@@ -73,11 +73,11 @@ The table below describes the various elements in this configuration file.
      - Y
      - Suggested plugin for reading the image files.
    * - Envelope2D
-     - Y
+     - N
```

Comment:
This option seems not to actually be required. In any case, it does not appear in the example configuration shown below, so either this line or the configuration is wrong. (That is, shouldn't the option appear in the example configuration if it is mandatory?)